oak_protobuf_idl/proto/messages.proto

```diff
@@ -0,0 +1,45 @@
+//
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak.protobuf_idl;
+
+// A message representing a request over a transport.
+message Request {
+  uint32 method_id = 1;
+  bytes body = 2;
+}
+
+// A message representing an error status code with associated message.
+//
+// Similar to https://github.com/googleapis/googleapis/blob/master/google/rpc/status.proto.
+message Status {
+  // The status code, which should be an enum value of
+  // https://github.com/googleapis/googleapis/blob/master/google/rpc/code.proto.
+  int32 code = 1;
```

Comment:
Is it not possible to import and use the enum as the type of this field?

Reply:
It seems to cause more problems than it solves, because of transitive imports etc. Perhaps that's also why gRPC does it like this, plus in principle the enum can evolve separately from this message, though I think that's unlikely to be useful in practice. Anyways, let's try it out as it is, but we can move things around later.

Reply:
I see. Thanks for clarifying :)